pkgs/development/interpreters/python/tests/test_python.py

```diff
@@ -46,6 +46,12 @@ def test_base_prefix(self):
     def test_python_version(self):
         self.assertTrue(platform.python_version().startswith(PYTHON_VERSION))
 
+    @unittest.skipIf(IS_NIXENV and IS_VENV, "TODO: Import from site-package does not work in nixenv-venv.")
```

Comment:
Todo or not supported? I'd say we should not support this, but at the same time, I know those that really like such feature.

Reply:
Hm, there are quite a few issues open with that, but then again i don't really know the exact use cases. For me it all started with that i wanted to build rust `servo`, this needs `mach` and this needs `psutil` and then i was in the situation that when having a `nix-shell --pure` with `python.withPackages` and `psutil`, the `mach` build would not use the  `psutil` from `python.withPackages` but trying to build a fresh one which does not succeed since `gcc` is not present because it's a pure shell and for `servo` to build it went with `clang`.

I don't even know if that counts as use case, because i don't even understand if the `mach` system is trying to use `venv` or `virtualenv` features, and i cannot understand why it is not using the prebuild `psutil` and thought it might be related.

Reply:
Said that, feel free to change it to your liking :)

Reply:
And on another thought, it just feels wrong if `venv` or `virtualenv` is present and usable in `python.withPackages` but one can not access the site-packages.

Reply:
At least `virtualenv` does have the option `--system-site-packages` from which i would expect that this is indeed possible.

Reply:
I'm running the tests against staging right now, and have a PR waiting against staging. I changed the wording in skipIf to be "Unsupported: ..."